server/services/event-handlers.js

```diff
@@ -21,9 +22,11 @@ const createRawCaptureFeatureHandler = (async (message) => {
   executeRawCreateCaptureFeature(newRawCaptureFeature);
 })
 
+
 const registerEventHandlers = () => {
   subscribe("capture-created", createCaptureFeatureHandler);
   subscribe("raw-capture-created", createRawCaptureFeatureHandler);
+  subscribe("token-assigned", tokenAssignedHandler);

```

Comment:
In the intergation test, we could choose to invoke `tokenAssginedHandler` directly to test this flow by not involving rabbitmq.